_sass/pages/_faq.scss

```diff
@@ -24,15 +30,42 @@
     margin: 40px 0 0 0;
   }
 
+  .list-unstyled {
+    li {
+      &:hover {
+        h6 {
+          color: $black;
+
+          &.arrow-d,
+          &.arrow-r {
+            &:before {
+              -webkit-transition: all .25s ease-in-out;
```

Comment:
You can add `@mixin` for this kind of transitions to not repeat same properties each time.

Reply:
Done